DOCS-8120 Spec Downloader Lightning Component (#146)
mulesoft/docs-api-community-manager: modules/ROOT/pages/acm-lightning-components.adoc

Registrations Require Approval:: Select whether registration requires approval.
Approver ID:: Specifies the approver ID.

> == Spec Downloader

Reviewer: Spec Downloader Versions and Options?

Author: On this page each header is the exact name of the component, such as "Spec Downloader", so I'm preserving the current text there.

> == Spec Downloader

Reviewer: ```suggestion
== Spec Downloader Component
```

> The Spec Downloader component enables users to download an API specification.
> 
> The API specification downloaded from API Community Manager is the same as the specification downloaded from Anypoint Exchange.

Reviewer: How about...:
```suggestion
The Spec Downloader component enables users to download an API specification.

The Spec Downloader component <of this> enables users to download an API specification from API Community Manager that is the same as the Anypoint Exchange specification. In <this UI page>, users can click either of the following:
* *Download Spec as RAML* +
* *Download Spec as OAS* +
```
...but I can't figure out what the rest means. Is the general practice to download to the same type (RAML or OAS) as originally uploaded? And if the two differ, then "the downloaded version is automatically generated"? What does that mean, that it is automatically generated; what is the alternative? And why would an automatically generated version be different from its source?

If this is clear to your intended audience, then ignore my confusion. But look at this text again and be sure that the wording you use "connects the dots" and does not rely on audience assumptions.

Author: This is explained in more detail in the Exchange docs, and will be somewhat clearer to the audience, but it's a good point, and I added some more explanation. Users can upload either type, and the other type will be generated automatically so that both are available. The alternative would be for one of them to be missing. The differences from the source are caused by unavoidable differences in the RAML and OAS formats.

> The available buttons are *Download Spec as RAML* and *Download Spec as OAS*. If the API specification is uploaded as RAML and downloaded as OAS, or uploaded as OAS and downloaded as RAML, then the downloaded version is automatically generated and may have small inconsistencies with the uploaded version.

Reviewer: See above.

Reviewer: may > might

Author: Good point. Fixed.

> You can specify values for the following parameters of the Spec Downloader component:

Reviewer: So, after the user chooses the download type, the user customizes using parameters?
Or is this what the admin does?
How are these parameters related to the previous text about the buttons? Do we need a screenshot?
If these are UI labels, they should be formatted as such (* *).

Author: ACM uses Salesforce Community Cloud, which is an interactive web page for building interactive web pages. The admin uses ACM to drag and drop the components listed on this page into a page for developer users and then clicks "publish" to build it. So the admin will drag and drop a Spec Downloader onto the page, and specify the Record Id and Layout options, and the users will just see buttons to click: the *Download Spec as RAML* or *Download Spec as OAS* buttons. "Vertical" and "Horizontal" are options in a drop-down menu.

Record Id:: Specifies the API. Default is `{!recordId}`, which downloads the specification of the API selected on the current page.

> Layout:: Specifies the layout of the buttons, `Horizontal` or `Vertical`. Default is `Horizontal`.

Reviewer: ```suggestion
Layout:: Specifies the layout of the buttons, `Horizontal` (default) or `Vertical`.
```

Author: Good point! Fixed.

> == Filter Conditions [[filter-conditions]]

Reviewer: ```suggestion
== Filter Conditions Parameter [[filter-conditions]]
```

Author: This one is also an exact component name, so I'm preserving the current text.
